posydon/interpolation/IF_interpolation.py

```diff
@@ -359,7 +359,7 @@ def save(self, filename):
 class BaseIFInterpolator:
     """Class handling the initial-final interpolation for POSYDON."""
 
-    def __init__(self, grid=None, in_keys=None, out_keys=None, in_scaling=None,
+    def __init__(self, grid=None, in_keys=None, out_keys=None, out_nan_keys=None, in_scaling=None,
```

Comment:
Have you checked that later functions can handle a None as input for `out_nan_keys`? So far it got initialized as an empty list in case it wasn't set.

Reply:
This is fine, since it's set to a value in this function, if it's `None`. 
See [line 463.](https://github.com/POSYDON-code/POSYDON/blob/191f056e4fa35e9b63324cf8fffba5dfe03486c0/posydon/interpolation/IF_interpolation.py#L463)

Reply:
True, my question was, whether you checked, that all function calls are fine.
E.g. there is one in line 446: has this no problems, with changing the type from list to None-type.
After having a look at this function, yes it looks to be OK with this change.